plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Rename/Internal.hs

```diff
@@ -0,0 +1,36 @@
+-- | The internal module of the renamer that defines the actual algorithms,
+-- but not the user-facing API.
+
+module Language.UntypedPlutusCore.Rename.Internal
+    ( module Export
+    , renameTermM
+    , renameProgramM
+    ) where
+
+import           Language.UntypedPlutusCore.Core
+
+import           Language.PlutusCore.Core         (HasUniques)
+import           Language.PlutusCore.Name
+import           Language.PlutusCore.Quote
+import           Language.PlutusCore.Rename.Monad as Export
+
+
+-- | Rename a 'Term' in the 'RenameM' monad.
+renameTermM
+    :: (HasUniques (Term name uni ann), MonadQuote m)
+    => Term name uni ann -> ScopedRenameT m (Term name uni ann)
```

Comment:
Hm. We don't need `ScopedRenameT` as it's for cases when we have two scopes (the type-level and term-level ones). We don't have the type-level scope, so here we should be able to use just `RenameT` (as when we rename types in Plutus Core where we also have just one scope).

Not that it's too important, though, and we can do it later.